share/isso.conf

```diff
@@ -71,6 +71,10 @@ password = please_choose_a_strong_password
 # them.
 enabled = false
 
+# with moderation enabled, automatically approve new posts by a
+# previously approved author
+activate-if-email-previously-approved = false
```

Comment:
Should this be called approve rather than activate? The term activate is a bit confusing to me (but I'm not a native English speaker).

Reply:
Please also add documentation for this new option.

Reply:
Hey there @jelmer I agree with you on approve vs activate. I chose activate, because that's the wording used in the moderation emails. It sounds like you prefer "approve" so I'll change it, OK?

(P.S. we hadden dit ook in het Nederlands kunnen bespreken, maar dat is niet zo leuk voor de rest. ;)

Reply:
I will add some documentation to the `server.rst` and push within about 10 minutes.